.github/CODEOWNERS

```diff
@@ -48,6 +48,15 @@
 # Module: backend-proxy-middleware
 /packages/backend-proxy-middleware/ @SAP/ux-tools-middlewares-and-tasks
 
+# Module: preview-middleware
+/packages/preview-middleware/ @SAP/ux-tools-middlewares-and-tasks @SAP/ux-tools-adp
```

Comment:
Do these teams exist?
And have write access to the repo?
Otherwise they are not used.

Reply:
yes and yes